Python/Examples/include/ActsPython/Examples/Helpers.hpp

```diff
@@ -0,0 +1,20 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Framework/AlgorithmContext.hpp"
+#include "ActsExamples/Framework/ProcessCode.hpp"
+
+namespace ActsPython::Concepts {
+template <typename T>
+concept has_write_method =
+    requires(T& t, const ActsExamples::AlgorithmContext& ctx) {
+      { t.write(ctx) } -> std::same_as<ActsExamples::ProcessCode>;
+    };
+}  // namespace ActsPython::Concepts
```

Comment:
In my opinion it's questionable if this needs to be in a separate file if it's only used in `Macros.hpp` in one location.

Reply:
I've put it into a different file, because this one has Framework dependencies (I can get around within the macros without dependency).